multichain-testing/pool-config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "weights": "1uosmo,1ibc/E7827844CB818EE9C4DB2C159F1543FF62B26213B44CE8029D5CEFE52F0EE596",
```

Comment:
What is `ibc/E7827844CB818EE9C4DB2C159F1543FF62B26213B44CE8029D5CEFE52F0EE596` ?

Let's maybe rename this file `osmo-[bld?]-pool-config.json`

Reply:
Request addressed on commit [c521264](https://github.com/Agoric/agoric-sdk/pull/11229/commits/c521264e960c88c85f428bfbff5de36f2af2d6ca)

The IBC denom referred to is ubld. 

I concluded that the pool-config.json file doesn’t need to remain permanently in the agoric-sdk repo, since it is automatically generated by the create-osmosis-pool.sh script. With that in mind, I removed the file and added it to .gitignore to prevent future accidental commits.

The file is now being created with the name structure you suggested.